config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 7b4d22d602952a1cda784da2bb84425f4d32933baee420f731265da8fc4d2613b90d7e1d274c2b39ac6b13f551102c205dbfa46e229460268a7755280816c8aa
+
+test:
+  secret_key_base: 8a4bc7377c3b247399cee288ae5df59cb73e3b9124bd4d85dcadd2852a1738dbd9b5bd8b849c652556c7ab8b456b6cfe4b4e99feafb256575bcdfdde54177a73
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
+production:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```

Comment:
make sure when you deploy

You should add secrets.yml to git (i.e. remove it from .gitignore) after replacing all sensitive keys with references to environment variables.

So:
```
production:
  secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
```

then run: 

```
heroku config:set SECRET_KEY_BASE ='7658699e0f765e8whatever'
```

And then make sure you add it to gitignore, so changes in the file won't be suggested to be added to a commit by git anymore